public/app/plugins/panel/state-timeline/utils.ts

```diff
@@ -154,7 +145,6 @@ export const preparePlotConfigBuilder: UPlotConfigPrepFn<TimelineOptions> = ({
     values: coreConfig.yValues,
     grid: false,
     ticks: false,
-    size: yAxisWidth,
```

Comment:
@leeoniya by removing this,  calculateAxisSize is now called and it handled the same measureText bit that we use for normal time series Y axis ticks  https://github.com/grafana/grafana/blob/main/packages/grafana-ui/src/components/uPlot/config/UPlotAxisBuilder.ts#L130

Reply:
nice, i hated this hack.